src/lib/types.hpp

```diff
@@ -132,7 +132,7 @@ using TaskID = uint32_t;
 using CommitID = uint32_t;
 using TransactionID = uint32_t;
 
-using AttributeVectorWidth = uint8_t;
+using VectorCompressionID = uint8_t;
```

Comment:
A comment explaining that the ID represents the corresponding `CompressedVectorType` would have helped me to understand what this value is used for. The naming is not clear to me (see above).

Reply:
changed the name to `CompressedVectorTypeID`, so it should hopefully be clear now. I'd avoid an unnecessary comment stating that it is the ID for the CompressedVectorType.